docs/sql/statements/start-transaction.rst

```diff
@@ -0,0 +1,42 @@
+.. highlight:: psql
+.. _ref-start:
```

Comment:
```suggestion
.. highlight:: psql

.. sql-start-transaction:
```

this is the new recommended style for RST labels. not a lot of files use this yet though but it's good to get it in at the beginning for new docs

Reply:
how do I reference this new syntax? `.. sql-start-transaction:`  I tried :ref:`START <ref-sql-start-transaction>` but does not work ..

Reply:
the entire label is opaque (from a syntax perspective) so the `ref` prefix doesn't mean anything special. to reference a label named `sql-start-transaction` you would use this syntax:

    :ref:`example <sql-start-transaction>`

Reply:
oops my bad I just noticed I accidentally missed the initial `_` character

my original suggested change was:

    .. sql-start-transaction:

my corrected suggestion is:

    .. _sql-start-transaction: